src/VisualStudio/Setup.ServiceHub/Roslyn.VisualStudio.Setup.ServiceHub.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <RoslynProjectType>Custom</RoslynProjectType>
+
+    <!-- VS Insertion -->
+    <TargetVsixContainerName>Roslyn.VisualStudio.Setup.ServiceHub.vsix</TargetVsixContainerName>
+    <VisualStudioInsertionComponent>Roslyn.VisualStudio.Setup.ServiceHub</VisualStudioInsertionComponent>
```

Comment:
>Roslyn.VisualStudio.Setup.ServiceHub [](start = 36, length = 36)

This should be `Microsoft.CodeAnalysis.LanguageServices` - we'll include the VSIX in our existing component

Reply:
okay